analysed-packages/webrtcHacks-adapter/version-7.3.0/README.md

```diff
@@ -0,0 +1,17 @@
+## Download Location
+
+https://github.com/webrtcHacks/adapter/archive/v7.3.0.tar.gz
+
+## Package URL (purl)
+
+pkg:github/webrtcHacks/adapter@v7.3.0
+
+## Creator
+
+Sascha Dierberg <dierberg@dresearch-fe.de>
+
+## Reviewers
+
+The information was reviewed by:
+
+* [Reviewer]
```

Comment:
The reviewer in the readme.md is missing. Everything else is looking fine.

Reply:
Mario Schuknecht added as reviewer